Include/internal/pycore_code.h

```diff
@@ -257,6 +257,20 @@ extern void _PyStaticCode_Fini(PyCodeObject *co);
 extern int _PyStaticCode_Init(PyCodeObject *co);
 
 /* Tier 2 interpreter */
+
+// gen_bb_is_successor:
+//   true = successor
+//   false = alternate
+// gen_bb_requires_pop:
+//   For tier2 type propagation, handling of jump instructions with
+//   runtime-dependent stack effect.
+//   This flag is used to determine if the type context of a new bb
+//   requires a stack element to be popped.
+#define BB_TEST(gen_bb_is_successor, gen_bb_requires_pop) \
+    (((gen_bb_is_successor) << 1) + (gen_bb_requires_pop))
```

Comment:
Micro optimisation: use bitwise or
```suggestion
    (((gen_bb_is_successor) << 1) | (gen_bb_requires_pop))
```

Reply:
Also this is the wrong file. It should be in `ceval_macros.h`